community.md

```diff
@@ -292,6 +292,7 @@ You can also find cool projects or submit your own to the [faas-and-furious orga
 
 | Project name and description                                         | Author     | Site      | Date        |
 |----------------------------------------------------------------------|------------|-----------|-------------|
+| [Faasflow - faas workflow as a function](https://github.com/s8sg/faasflow) | Swarvanu Sengupta | github.com | 11-Nov-2018 |
```

Comment:
Might be worth renaming the repo to `faas-flow` to match the naming convention used with other third-party OpenFaaS integrations.

Reply:
It also acts as a golang package. As I remember faas-flow is not a valid package name